pkg/object/object.go

```diff
@@ -70,10 +75,30 @@ type Volume struct {
 	StorageClass *string
 }
 
+type Secret struct {
+	Name string
+	Data []SecretData
+}
+
+type SecretData struct {
+	Key       string
+	Plaintext *string
+	Base64    *string
+	File      *string
+}
+
 type OpenCompose struct {
 	Version  string
 	Services []Service
 	Volumes  []Volume
+	Secrets  []Secret
+}
+
+type Input struct {
```

Comment:
Added comments to explain this.
As for it being a part of object.go, I'd ideally want this to be a part of the `cmd` package, but then that messes up the go import cycle when this is used in `encoding.go`, so I made this a part of `object.go`